src/common/ActivityInfo/ActivityInfo.tsx

```diff
@@ -20,18 +20,25 @@ export interface SocialInfo {
   link: string;
 }
 
-export interface InfoProps {
-  title: string;
+export interface ActivityInfoProps {
+  name: string;
   id: string;
-  description: string;
-  startTime: Date;
-  endTime: Date;
-  speakerName: string;
-  speakerAbout: string;
-  speakerSocials: SocialInfo[];
-  relatedActivities?: string[];
+  eventId: string;
+  startTime: string;
+  endTime: string;
+  duration: number;
+  mediaType: "embed_url" | "meeting_url" | "embed_url_require_auth";
+  mediaLink: string;
+  thumbnail: string;
+  presenter: string;
+  presenterAbout: string;
+  presenterSocials: SocialInfo[];
```

Comment:
you don't like my variable names??????????? :'(

Reply:
it was either use your variable names, or change these 3 names in all the .md files and in the heat of the moment I opted for the easier option. no hard feelings.
![74a9f5341f5bbaa0c77050c10c59d4ae](https://user-images.githubusercontent.com/34114190/115611137-555bfb00-a2af-11eb-85c7-ef41ca402fc7.jpg)